django-backend/fecfiler/scha_transactions/views.py

```diff
@@ -29,6 +30,24 @@ class SchATransactionViewSet(CommitteeOwnedViewSet):
     ).all()
     """QuerySet: all schedule a transactions with an aditional contributor_name field"""
 
+    def get_queryset(self):
+        assert self.queryset is not None, (
```

Comment:
since this class inherits `CommitteeOwnedViewSet`, use self.get_queryset() at line